src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletServiceImpl.java

```diff
@@ -22,7 +22,7 @@ public class WalletServiceImpl implements WalletService {
 
     @Override
     @Transactional
-    public WalletDTO updateWallet(EditWalletDto walletToEdit) throws WalletNotFoundException {
+    public WalletDTO updateWallet(UpdateWalletDto walletToEdit) throws WalletNotFoundException {
```

Comment:
not `UpdateWalletDTO walletToUpdate`
but `UpdateWalletDTO walletToUpdate`

Reply:
done